challenges/2021_cloud_base_height/03_pytorch_train.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        trainer_hparams = {
</pre><p>maybe put the definitions in a separate cell so this one isnt so huge</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I found that a lot of the objects depend on having parameters from the mlflow run (which is started with the `with` keyword) and feel that the training loop is difficult to reduce as it is the main element of this notebook. I have tried to clean up where possible though